docs/using.rst

```diff
@@ -4,21 +4,39 @@
  Using :mod:`!importlib_metadata`
 =================================
 
-``importlib_metadata`` is a library that provides access to installed
-package metadata, such as its entry points or its
-top-level name.  Built in part on Python's import system, this library
+``importlib_metadata`` is a library that provides access to
+the metadata of an installed :term:`packaging:Distribution Package`,
+such as its entry points
+or its top-level :term:`packaging:Import Package` name.
```

Comment:
This phrasing is misleading, suggesting that a distribution package has exactly one import package, when a distribution package may have any number of package names (including none) and may also contain names of modules supplied by the distribution package. Maybe just "top-level names" would be more accurate.

Reply:
```suggestion
or its top-level names (:term:`packaging:Import Package`s, modules, if any).
```